app/ui/browser/model/index.js

```diff
@@ -52,4 +54,8 @@ export const Page = Immutable.Record({
   canRefresh: false,
   chromeMode: 'expanded',
   commands: Immutable.List(),
-});
+}) {
+  constructor(data) {
+    super(Object.assign({ id: uuid.v4() }, data));
```

Comment:
@rnewman here's where we create a uuid -- we can [configure the options](https://www.npmjs.com/package/uuid#uuidv4options--buffer--offset) to whatever and move this into a shared util used by all processes